### ISSUE TITLE
Lack of information about the Consume Flag

### ISSUE THREAD

**Issue by SRechenberger** [purescript-contrib/purescript-parsing#68]
The documentation of [Parser.purs](https://github.com/purescript-contrib/purescript-parsing/blob/master/src/Text/Parsing/Parser.purs) lacks information about the third parameter of `ParseState` and what setting it to `True` does.

**Comment by jamesdbrock** [comment]
Yeah, tell me about it. We need documentation for the consumed flag.
